ansibullbot/triagers/plugins/component_matching.py

```diff
@@ -142,6 +145,14 @@ def get_component_match_facts(iw, component_matcher, valid_labels):
                 if u'/plugins/' in x:
                     cmeta[u'is_new_plugin'] = True
 
+    # is it a colleciton?
+    if [x for x in CM_MATCHES if x[u'repo_filename'].startswith(u'collection:')]:
```

Comment:
Better avoid creating in-mem lists. This will also save time by stopping the iteration early:
```suggestion
    is_collection = any(x[u'repo_filename'].startswith(u'collection:') for x in CM_MATCHES)
    if is_collection:
```

Reply:
Alternatively, you could save this list in a var since you use it later in code. So that you'd not repeat this twice:
```suggestion
    collection_components = [x for x in CM_MATCHES if x[u'repo_filename'].startswith(u'collection:')]
    if collection_components:
```
Then, you could reuse this var on line 153